Resource Server/controllers/passport.js

```diff
@@ -12,27 +14,50 @@ passport.use(
       callbackURL: process.env.GOOGLE_CALLBACK_URL,
     },
     async function (request, accessToken, refreshToken, profile, done) {
-      // TODO: Add user handling logic here
+      try {
+        const user = await dbQueries.getUserById([profile.email]);
 
-      // Continue with authentication flow
-      return done(null, profile);
+        if (!user) {
+          await dbQueries.insertUser([profile.email, 0]);
```

Comment:
To insert a new user you only need to email as I set it to always insert 0 for the score